src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCF.java

```diff
@@ -224,51 +262,138 @@ protected void createAnnotationEngine() {
     // get VariantContexts from input gVCFs and regenotype
     @Override
     public void apply(VariantContext variant, ReadsContext reads, ReferenceContext ref, FeatureContext features) {
-        final VariantContext newVC = regenotypeVC(variant);
-        if (newVC != null) {
-            vcfWriter.add(newVC);
-        }
+        regenotypeVC(variant);
+    }
+
+    @Override
+    public Object onTraversalSuccess() {
+        vcfWriter.close();
```

Comment:
`vcfWriter` is later closed in `closeTool`, so this leads to throwing an exception

Reply:
Yeah, that was a tough one to figure out, but I fixed that locally already.